.pep8speaks.yml

```diff
@@ -2,7 +2,7 @@
 # File : .pep8speaks.yml
 
 scanner:
-    diff_only: True  # If False, the entire file touched by the Pull Request is scanned for errors. If True, only the diff is scanned.
+    diff_only: False  # If False, the entire file touched by the Pull Request is scanned for errors. If True, only the diff is scanned.
```

Comment:
Why this? I believe it would make PEP8Speaks slow? Just curious.

Reply:
Because if you see the change in setup.py in this PR, it's a minor PEP8 fix which made me realize that while the code committed from my side would be compliant, there's no guarantee that other parts of them do not contain mistakes.

This would help eventually take care of any past non compliance that happens as the files continue to be worked upon, and I'm sure the speed difference would be negligible enough to make a difference.

Reply:
Hmm, so for that what I would suggest is that run PEP8SPeaks on the entire code once, fix all bugs and thereafter make it run on only diff and accept new code through PRs only.

Does this sound better? :thinking:

Reply:
That's what I thought, but pep8speaks does not give you that option. So then I thought I can run it locally and then fix if any issues. But that doesn't address the code not committed by me unless the PR only policy is enforced which is nice but sometimes slower. So then I thought let's just add flake8 to travis ez, but then would have needed to config it 😂 
So ultimately this was the laziest and most concise way to fix it long term that I could think of 😛

Reply:
Cool. Makes sense if you want to make the contributions faster :)